packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -259,15 +259,23 @@ function buildHelper(
 
 function buildRuntimeRewritePlugin(runtimeName, helperName) {
   /**
-   * rewrite helpers imports to runtime imports
+   * Rewrite helper imports to load the adequate module format version
    * @example
    * adjustImportPath(ast`"setPrototypeOf"`)
-   * // returns ast`"@babel/runtime/helpers/esm/setPrototypeOf"`
-   * @param {*} node The string literal contains import path
+   * // returns ast`"./setPrototypeOf"`
+   * @example
+   * adjustImportPath(ast`"@babel/runtime/helpers/typeof"`)
```

Comment:
The `import "@babel/runtime/helpers/typeof"` is generated by `@babel/plugin-transform-typeof-symbol` which runs before the `buildRuntimeRewritePlugin` plugin. In this case I will prefer just convert `"@babel/runtime/helpers/typeof"` to `"./typeof"`: So this plugin does not accidentally support `"@babel/runtime/helpers/another-helper"`.

Reply:
If there are injected imports for other helpers, wouldn't we want to transform them too?

Reply:
Yes they should.

Not related to this PR: I realized that the generated helper will target to browserlists's default in Babel 8 since we didn't supply a target in the `build-dist`. I guess it's fine to bump the targets but we should convey that in the changelog.